frontend/src/features/manageVehicles/components/dashboard/ManageVehiclesDashboard.tsx

```diff
@@ -52,21 +51,37 @@ export const ManageVehiclesDashboard = memo(() => {
     staleTime: staleTime,
   });
 
+  const powerUnitTypesQuery = useQuery({
+    queryKey: ["powerUnitTypes"],
+    queryFn: () => getPowerUnitTypes(),
+    keepPreviousData: keepPreviousData,
+    staleTime: staleTime
+  })
+
+  const trailerTypesQuery = useQuery({
+    queryKey: ["trailerTypes"],
+    queryFn: () => getTrailerTypes(),
+    keepPreviousData: keepPreviousData,
+    staleTime: staleTime
+  })
+
```

Comment:
I think we already have hooks for these, in "src/features/manageVehicles/apiManager/hooks.ts", the usePowerUnitTypesQuery and useTrailerTypesQuery

Reply:
You can reuse those hooks instead